addons/binding/org.openhab.binding.kostalinverter/ESH-INF/thing/thing-types.xml

```diff
@@ -38,7 +48,77 @@
 			</parameter>
 		</config-description>
 	</thing-type>
-
+	
+	<channel-type id="str1Voltage">
+	   <item-type>Number</item-type>
```

Comment:
As the binding already uses UoM we should change the item types to deliver a [QuantityType](https://www.openhab.org/docs/concepts/units-of-measurement.html#list-of-units).

```suggestion
	   <item-type>Number:ElectricPotential</item-type>
```